www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.xml

```diff
@@ -1,13 +1,14 @@
 <model>
     <mount>//Pischem/caddy</mount>
-    <description>A GUI model for configuring a reverse proxy in the Caddy web server.</description>
-    <version>1.2.2</version>
+    <description>Model for Caddy HTTPS and Layer4 Reverse Proxy.</description>
```

Comment:
to be honest you can keep this as simple as "Caddy Proxy". I don't think it's used much or needs explaining here.

Reply:
Changed description.